images/1.7.6.3/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM prestashop/base:5.6-apache
+LABEL maintainer="Thomas Nabord <thomas.nabord@prestashop.com>"
+
+ENV PS_VERSION 1.7.6.3
+
+# Get PrestaShop
+ADD https://www.prestashop.com/download/old/prestashop_1.7.6.3.zip /tmp/prestashop.zip
```

Comment:
Woah 😮 I have no idea what is this folder https://www.prestashop.com/download/old/

Reply:
it was automated by `generateAll.sh`